### PR TITLE
Add Memcached and PostgreSQL app doc

### PR DESCRIPTION
**Pull request by yunkunrao**

### CODE REVIEW

File: content/en/docs/application-store/built-in-apps/postgresql-app.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: "PostgreSQL App"
+keywords: 'Kubernetes, KubeSphere, PostgreSQL, app-store'
+description: 'How to use built-in PostgreSQL'
+
+
+weight: 2242
+---
+PostgreSQL is a powerful, open source object-relational database system which is famous for reliability, feature robustness, and performance. This guide will show you one-click deployment for PostgreSQL in Kubenetes .
+
```
Comment (FeynmanZhou) [inline content/en/docs/application-store/built-in-apps/postgresql-app.md:10]: The images are broken: https://deploy-preview-407--kubesphere-v3.netlify.app/docs/application-store/built-in-apps/postgresql-app/

File: content/en/docs/application-store/built-in-apps/memcached-app.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+2. Check app info and click `Deploy` button.
+
+![deploy_minio](/images/docs/appstore/memcached/deploy_memcached.png)
+
+3. Select app version and deployment location, then go to **Next → Deploy**
+
+![deploy_memcached_confirm](/images/docs/appstore/memcached/deploy_memcached_confirm.png)
+
+4. Wait for a few minutes, then you will see the application memcached showing active on the application list.
+
```
Comment (FeynmanZhou) [inline content/en/docs/application-store/built-in-apps/memcached-app.md:33]: Could you add a simple guide for how to connect and use Memcached on KubeSphere?

File: content/en/docs/application-store/built-in-apps/memcached-app.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+---
+title: "Memcached App"
+keywords: 'Kubernetes, KubeSphere, Memcached, app-store'
+description: 'How to use built-in Memcached Object Storage'
+
+
+weight: 2242
+---
+Memcached is designed for large data caches. Its API is available for most popular languages. This guide will show you one-click deployment for Memcached in Kubenetes .
```
Comment (FeynmanZhou) [inline content/en/docs/application-store/built-in-apps/memcached-app.md:9]: Add an external link with Memcached website here.

File: content/en/docs/application-store/built-in-apps/postgresql-app.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+---
+title: "PostgreSQL App"
+keywords: 'Kubernetes, KubeSphere, PostgreSQL, app-store'
+description: 'How to use built-in PostgreSQL'
+
+
+weight: 2242
+---
+PostgreSQL is a powerful, open source object-relational database system which is famous for reliability, feature robustness, and performance. This guide will show you one-click deployment for PostgreSQL in Kubenetes .
```
Comment (FeynmanZhou) [inline content/en/docs/application-store/built-in-apps/postgresql-app.md:9]: Add an external link with the PostgreSQL website here.

File: content/en/docs/application-store/built-in-apps/postgresql-app.md (hunk 1 above)
Comment (yunkunrao) [reply content/en/docs/application-store/built-in-apps/postgresql-app.md:10]: Got it. These broken links will be fixed after op doc is updated.

File: content/en/docs/application-store/built-in-apps/memcached-app.md (hunk 3 above)
Comment (yunkunrao) [reply content/en/docs/application-store/built-in-apps/memcached-app.md:9]: Got it

File: content/en/docs/application-store/built-in-apps/postgresql-app.md (hunk 4 above)
Comment (yunkunrao) [reply content/en/docs/application-store/built-in-apps/postgresql-app.md:9]: Got it

File: content/en/docs/application-store/built-in-apps/memcached-app.md (hunk 2 above)
Comment (yunkunrao) [reply content/en/docs/application-store/built-in-apps/memcached-app.md:33]: sure

**Review by rayzhou2017 — APPROVED** [review]

**Pull request merged**